Documentation/WhenToUseBabylonNative.md

```diff
@@ -0,0 +1,194 @@
+# When to Use Babylon Native
+
+## Questionnaire
+
+For each of the following questions, click on the link you feel best 
+represents your answer. The link will take you to another view of this
+same page with the next question for you at the top of your view. 
+Eventually, a link will take you to a lower section discussing whether
+or not we believe a Babylon Native technology might be right for your
+scenario and why.
+
+***
+
+#### Are you picking a platform for a new app, or is your app's platform already decided?
+-   [I'm picking a platform for my new app.](#do-you-prefer-more-native-like-technologies-or-more-web-like-technologies)
+-   [I'm adding 3D to an app that's already committed to a platform.](#is-your-app-built-on-react-native-another-javascript-using-platform-including-web-or-hybrid-app-platforms-or-non-javascript-technologies)
+
+#### Do you prefer more native-like technologies or more Web-like technologies?
+-   [I prefer more native-like technologies.](#do-you-have-exceptional-rendering-requirements-such-as-aaa-quality-graphics-or-extreme-performance-needs)
+-   [I prefer more Web-like technologies.](#do-you-already-have-existing-expertise-andor-preferences-for-react-native-or-do-you-need-native-capabilities-that-arent-provided-by-normal-web-apis-such-as-ar-on-ios)
+
+#### Do you have exceptional rendering requirements such as AAA-quality graphics or extreme performance needs?
+-   [I want things to look good, but I don't need extreme graphics or performance.](#babylon-native-is-probably-a-good-choice-for-your-project)
+-   [I need AAA-quality graphics and/or exceptional performance.](#there-may-be-other-choices-better-suited-to-your-project-than-babylon-native-technologies)
+
+#### Do you already have existing expertise and/or preferences for React Native, or do you need native capabilities that aren't provided by normal Web APIs (such as AR on iOS)?
+-   [Yes.](#babylon-react-native-is-probably-a-good-choice-for-your-project)
+-   [No.](#there-may-be-other-choices-better-suited-to-your-project-than-babylon-native-technologies)
+
+#### Is your app built on React Native, another JavaScript-using platform (including Web or hybrid app platforms), or non-JavaScript technologies?
+-   [My app is a React Native app.](#babylon-react-native-is-probably-a-good-choice-for-your-project)
+-   [My app's platform uses JavaScript, but it isn't made with React Native.](#there-may-be-other-choices-better-suited-to-your-project-than-babylon-native-technologies)
+-   [My app's platform doesn't inherently use JavaScript.](#do-you-have-exceptional-rendering-requirements-such-as-aaa-quality-graphics-or-extreme-performance-needs)
+
+***
+
+## Babylon Native is probably a good choice for your project!
+
+Congratulations! If you ended up here, then we think Babylon Native may
+be a suitable choice for adding 3D to your project. Typically, 
```

Comment:
We shouldn't assume 3D. Babylon Native can be used for scenarios that require access to the GPU.